package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "server-sdk-node",
+  "version": "1.0.1",
+  "description": "FeatureProbe Server Side SDK for Node.js",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "test:cov": "jest --coverage --no-cache --runInBand",
+    "lint": "eslint ./",
+    "build": "npm run build:ts && npm run build:web",
+    "build:ts": "tsc",
+    "build:web": "rollup -c rollup.config.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FeatureProbe/server-sdk-node.git"
+  },
+  "keywords": [
+    "featureprobe",
+    "server"
+  ],
+  "author": "FeatureProbe",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/FeatureProbe/server-sdk-node/issues"
+  },
+  "homepage": "https://github.com/FeatureProbe/server-sdk-node#readme",
+  "devDependencies": {
+    "@rollup/plugin-buble": "^0.21.3",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-typescript": "^8.3.2",
+    "@types/jest": "^29.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.31.0",
+    "eslint": "^8.20.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-check-file": "^1.2.2",
+    "eslint-plugin-import": "^2.26.0",
+    "jest": "^28.1.3",
+    "jest-environment-jsdom": "^29.0.1",
+    "jest-fetch-mock": "^3.0.3",
+    "pino": "^8.5.0",
+    "prettier": "^2.7.1",
+    "rollup": "^2.70.2",
+    "rollup-plugin-babel-minify": "^10.0.0",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-polyfills": "^0.2.1",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-serve": "^1.1.0",
+    "rollup-plugin-typescript2": "^0.31.2",
+    "ts-jest": "^28.0.8",
+    "typescript": "^4.7.4"
```

Comment:
Add typedoc as devDependency, by executing `npm run doc` to generate output
We will build those static assets later on our website